test/e2e/networks/ci.toml

```diff
@@ -55,34 +53,32 @@ retain_blocks = 3
 perturb = ["kill"]
 
 [node.validator04]
-persistent_peers = ["validator01"]
-database = "rocksdb"
 abci_protocol = "builtin"
-retain_blocks = 1
+database = "rocksdb"
+persistent_peers = ["validator01"]
 perturb = ["pause"]
+retain_blocks = 1
 
 [node.validator05]
+database = "cleveldb" 
+fast_sync = "v0" 
+seeds = ["seed02"] 
 start_at = 1005 # Becomes part of the validator set at 1010
-seeds = ["seed02"]
-database = "cleveldb"
-fast_sync = "v0"
 abci_protocol = "grpc"
 privval_protocol = "tcp"
 perturb = ["kill", "pause", "disconnect", "restart"]
 
 [node.full01]
-start_at = 1010
 mode = "full"
+start_at = 1010
 # FIXME Should use v1, but it won't catch up since some nodes don't have all blocks
 # https://github.com/tendermint/tendermint/issues/5444
 fast_sync = "v2"
 persistent_peers = ["validator01", "validator02", "validator03", "validator04", "validator05"]
 perturb = ["restart"]
 
 [node.full02]
-start_at = 1015
-mode = "full"
 fast_sync = "v2"
-state_sync = true
```

Comment:
Why did you disable state syncing for this node?

Reply:
Sorry it got a bit messed up when I was merging your most recent changes. Will correct it